src/shims/intrinsics.rs

```diff
@@ -438,20 +438,20 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 this.write_scalar(result_ptr, dest)?;
             }
 
-            "panic_if_uninhabited" |
-            "panic_if_zero_invalid" |
-            "panic_if_any_invalid" => {
+            "assert_inhabited" |
+            "assert_zero_valid" |
+            "assert_uninit_valid" => {
                 let ty = substs.type_at(0);
                 let layout = this.layout_of(ty)?;
                 if layout.abi.is_uninhabited() {
                     // Return here because we paniced instead of returning normally from the intrinsic.
```

Comment:
Heh, isn't it "panicked"? (at least according to my browser's spellchecker, "paniced" isn't a word and "panicked" is)

Reply:
https://github.com/rust-lang/miri/pull/1228